nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -272,6 +286,7 @@ in {
         default = null;
         description = ''
           The full path to a file that contains the database password.
+          Not used if <literal>createLocally</literal> is set.
```

Comment:
Not sure if I understand why it isn't used.

Also, please reference options with `<xref linkend="opt-services.nextcloud.foobar" />`.